### DISCUSSION TITLE
Preview images missing (videos/images, Largo, etc.) [self-hosted]

### DISCUSSION THREAD

**Discussion by fhandke-fugro** [biigle/core#828] [Q&A] [answered]
Hello,

I noticed that when I run a self-hosted instance on another PC and access it through the browser on my local machine all the little preview miniature images are missing:
![image](https://github.com/biigle/core/assets/78092148/57e1c66a-7f21-4b00-9c66-775e5da76362)

When I click on the images they are displayed as usual in full size after some loading.

If I open Biigle in the Browser of the server machine everything works and the preview images get displayed.

I guess it is connected to network throughput. Any idea on how I could try and fix this? Is there a setting allowing to wait longer and then have the preview images be displayed instead of defaulting to no display?

Thanks!

**Comment by mzur** [comment, marked as answer]
Please have a look at `APP_URL` in [build/.env](https://github.com/biigle/biigle/blob/2e88a538528bd62508083c8fe013994cee4d1a1f/build/.env.example#L9). Have you set it to `localhost`? It should be `http://<IP_ADDRESS>` with the IP address of the local server.

**Reply by fhandke-fugro** [reply]
That is awesome! Fixed it instantly. Thanks so much!

Now that I have you here maybe another question. Let me know if I should open another disucssion.

So when I perform Largo evaluation and want to delete all selected annotations it works fine. However when I want to relabel annotations and click on "Save" I get a "There was an unexpected error." message. It also appears with "Force delete/relabel":
![image](https://github.com/biigle/core/assets/78092148/f1ad2bd7-0164-42bc-866f-a521337ca22a)
Any idea here?

Thanks Martin

**Reply by mzur** [reply]
Please have a look at the "Logs" section of the Admin Area. You should find more detailed error messages there.

**Reply by fhandke-fugro** [reply]
Thanks! Figured it out. The video_annotation_label_feature_vectors table in the database was missing. The command "./artisan migrate" fixed it.
